### ISSUE TITLE
Shader Support (official shaders for forge)

### ISSUE THREAD

**Issue by szunakabeats** [FiniteReality/embeddium#256]
### Request Description

hey there, now i know this isnt maybe the right place to ask, but is there any good iris shader port for forge in sight ? oculus just isnt the thing ... it has so many bugs and just feels beeing let down. even with the latest updates shaders wont work. new visual bugs which werent there before arise and yea there isnt any good alternative besides oculus. maybe u have some contacts to the dev team from iris or sodium and could ask them to work with u together on this. i think the whole forge community is waiting on this.

anyways thx for your hard work and for your time reading this. have a nice week ! :-)

**Comment by embeddedt** [comment]
There indeed seems to be no great shader port for Forge right now. It's expected that Iris will be ported to (Neo)Forge in the near future. We're absolutely open to Iris offering support for Embeddium, but we don't expect this to happen as of right now, and honestly don't think it'd be realistic to expect the Iris team to support both Sodium *and* Embeddium. As we also don't forsee Iris dropping Sodium support, this sadly isn't very likely to happen.

The best chance of a better shader mod being available is for another developer to fork Iris for Embeddium compatibility. We'd obviously prefer to work with Iris directly, but as previously stated, we don't expect that to happen.

Offering shader support in Embeddium directly is not planned, as it's out of scope for the project (we focus on optimizing, not adding major new visual features).
